### ISSUE TITLE
`Eslint` complains of missing plugins when enabling `hardcore/ts` for `Vue` project

### ISSUE THREAD

**Issue by alexkramer98** [EvgenyOrekhov/eslint-config-hardcore#715]
Reproduction:

1. Create a new `Vue` project (TS).
2. Add `eslint` and `eslint-config-hardcore`.
3. Add the `.eslintrc.json` config from the `Typescript Vue 3` example.
4. Run `eslint src`.

```
Oops! Something went wrong! :(

ESLint: 8.36.0

ESLint couldn't find the plugin "eslint-plugin-etc".

(The package "eslint-plugin-etc" was not found when loaded as a Node module from the directory "/path/to/project".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-etc@latest --save-dev

The plugin "eslint-plugin-etc" was referenced from the config file in ".eslintrc.json » eslint-config-hardcore/ts#overrides[0]".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.

```
After installing the `eslint-plugin-etc` manually, the same error appears for `eslint-plugin-total-functions`. 
After installing this as well, everything works as expected.

The problem disappears when removing `hardcore/ts` from the `extends` list in `.eslintrc.json`. There is also no issue in `Nuxt 3` (both TS and JS).

I will have to debug this further. Meanwhile I am open to suggestions of what the problem may be.

**Comment by EvgenyOrekhov** [comment]
@alexkramer98 Can't reproduce. Could you try the latest version? eslint-config-hardcore@35.2.4

**Comment by arthureggert** [comment]
I have the same issue on my small local repo. All the hardcore/ts plugins are not installed.

**Comment by alexkramer98** [comment]
Can confirm the issue persists using the newest `eslint-config-hardcore`.

**Comment by alexkramer98** [comment]
On a new `Vue 3 (ts)` project, `eslint` complains of missing:

- `eslint-plugin-etc`
- `eslint-plugin-total-functions`

**Comment by orbachar** [comment]
same here

**Comment by EvgenyOrekhov** [comment]
Fixed and released in [**eslint-config-hardcore@36.0.0**](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v36.0.0)
